src/python/WMCore/MicroService/Unified/MSRuleCleaner.py

```diff
@@ -422,21 +426,18 @@ def cleanRucioRules(self, wflow):
         delResults = []
         if self.msConfig['enableRealMode']:
             for rule in wflow['RulesToClean'][currPline]:
-                msg = "%s: Deleting rule['id']: %s "
-                msg += "For did: %s"
-                self.logger.info(msg, currPline, rule['id'], rule['name'])
-                delResult = self.rucio.deleteRule(rule['id'])
+                msg = "%s: Deleting ruleId: %s "
```

Comment:
Please compact this `msg` and the line below altogether, such as:
```
self.logger.info("%s: Deleting ruleId: %s ", currPline, rule)
```

Line is not long enough and there is no need to break it in multiple lines.

Reply:
Done